WS2012R2/lisa/remote-scripts/ica/provisionLinuxForLisa.sh

```diff
@@ -276,24 +280,29 @@ function RhelTasks
 	# Disable the firewall
 	#
 	LogMsg "Info : Disabling the firewall"
-	
+
 	service iptables stop
 	chkconfig iptables off
-	
+
 	service ip6tables stop
 	chkconfig ip6tables off
-	
+
 	#
 	# Disable SELinux
 	#
 	LogMsg "Info : Disabling SELinux"
     sed -i '/^SELINUX=/cSELINUX=disabled' /etc/selinux/config
-	
+
 	#
 	# Create a list of packages to install, then ensure they are installed
 	#
 	installError=0
-    packagesToInstall=(at bridge-utils btrfs-progs crash dos2unix dosfstools e2fsprogs e2fsprogs-libs util-linux gpm dump system-config-kdump libaio-devel nano ntp ntpdate parted wget xfsprogs iscsi-initiator-utils bc)
+    if [ $1 -eq 7 ]; then
+	    packagesToInstall=(at bridge-utils btrfs-progs crash dos2unix dosfstools e2fsprogs e2fsprogs-libs util-linux gpm dump system-config-kdump libaio-devel nano ntp ntpdate parted wget xfsprogs iscsi-initiator-utils bc)
+    fi
+    if [ $1 -eq 6 ]; then
+	    packagesToInstall=(at bridge-utils btrfs-progs crash dos2unix dosfstools e2fsprogs e2fsprogs-libs util-linux gpm dump system-config-kdump libaio-devel nano ntp ntpdate parted wget iscsi-initiator-utils bc)
+    fi
```

Comment:
--> function RhelTasks is not updated to take parameters, it might just work as it is but it might not be the proper way to handle the release version tag.
Hi Chris, 
with RhelTasks 6 and RhelTasks 7 function, 6 and 7 is as the first parameter to take parameter.